setup.py

```diff
@@ -104,7 +104,7 @@
         "PyGithub>=1.54,<2",
         "pytest>=6.0,<8.0",
         "python-dateutil>=2.8.2,<3",
-        "pyyaml>=6.0,<7",
+        "PyYAML>=5.0,<7",
```

Comment:
i never realized these were case insensitive
some of the others still using casing tho

should we make all of them lower case or should we stick to the rule of using actual pypi names?
kinda nice to know what the actual pypi name is (with casing)

Reply:
Would use the pypi name, but can update later no biggie

Reply:
oh i see, no worries then